interchaintest/go.mod

```diff
@@ -10,6 +10,7 @@ replace (
 	// For this nested module, you always want to replace the parent reference with the current worktree.
 	github.com/CosmosContracts/juno/v19 v19.0.0-00010101000000-000000000000 => ../
 	github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.2 //indirect
+	github.com/cometbft/cometbft => github.com/cometbft/cometbft v0.37.2
```

Comment:
Howdy there, you're replacing comet with old-comet.

The purpose of the exercise was to use the latest dependencies, clearing issues that are solved in the latest comet like the super annoying btcd security warning.  

Thank you for conclusively showing that interchaintest is not kept up to date. 

cometbft's latest release is v0.37.4